rest_api/actix_web_1/Cargo.toml

```diff
@@ -27,6 +27,7 @@ actix-web = "1"
 futures = { version = "0.1" }
 log = "0.4"
 serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
```

Comment:
lint - should be "1"

Reply:
fixed, Should this be fixed in libsplitner?